src/elastio-snap.c

```diff
@@ -4594,14 +4595,17 @@ static int __tracer_destroy_cow(struct snap_device *dev, int close_method){
 		}
 	}
 	
-	if (dev->sd_cow_extents) {
+	if (close_method != 2 && dev->sd_cow_extents) {
```

Comment:
If we remove the cow file while dorming, preserve the cow file extents. They will be freed when the `close_method` will be 0 (final module unload)